components/automate-ui/src/app/pages/+compliance/+profile/profile.routing.ts

```diff
@@ -6,16 +6,11 @@ import { ProfileOverviewComponent } from './+profile-overview/profile-overview.c
 const routes: Routes = [
   {
     path: '',
-    children: [
-        {
-        path: '',
-        component: ProfileOverviewComponent
-        },
-        {
-        path: 'profile-details',
-        component: ProfileDetailsComponent
-        }
-    ]
+    component: ProfileOverviewComponent
+  },
+  {
+    path: 'profile-details',
+    component: ProfileDetailsComponent
```

Comment:
This is just a simplification, I suppose? Or has it any consequence?

Reply:
this url is compliance/compliance-profiles/profile-details...
The /profile-details is the same and hasn't been changed and the /compliance-profiles has been updated from /profiles.